lib/joy.ml

```diff
@@ -1,6 +1,6 @@
 let context = Context.context
 
-type point = Shape.point
+type 'a point = 'a Shape.point
```

Comment:
What does this type change aim to do?

Reply:
Make it easier to convert between/handle the `int point`s received from the end user